### PR TITLE
Add !blackjack command

### PR DESCRIPTION
**Pull request by maximilianspitzer**
Added a !blackjack command to main list, and changed some help texts to reflect changes. 

This is useful (imo) because ive had a lot of people explicitly ask me to rig blackjack and also seen them play the game in a non-optimal way. I added instructions to make ~50% more money over time.

### CODE REVIEW

**Comment by maximilianspitzer** [comment]
Sidenote: Im not sure why the !fix alias is still in this PR, even though there is no change to the file.

File: ChatCommander.lua [hunk 1]
```diff
@@ -444,6 +456,8 @@ local function afk_casino_tick()
         force_roulette_area()
         force_rig_roulette()
         force_rig_roulette()
+        force_rig_blackjack()
+        force_rig_blackjack()
```
Comment (maximilianspitzer) [inline ChatCommander.lua:460]: Also would like to know why calling the function twice per tick is done here @hexarobi ?

File: lib/ChatCommands/vehicle/repair.lua [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+            VEHICLE.SET_VEHICLE_FIXED(vehicle)
+            -- Also repair vehicle if its been destroyed by water
+            VEHICLE.SET_VEHICLE_DIRT_LEVEL(vehicle, 0.0)
+            VEHICLE.SET_VEHICLE_UNDRIVEABLE(vehicle, false)
+            VEHICLE.SET_VEHICLE_ENGINE_HEALTH(vehicle, 1000.0)
+            VEHICLE.SET_VEHICLE_PETROL_TANK_HEALTH(vehicle, 1000.0)
+            VEHICLE.SET_VEHICLE_ENGINE_ON(vehicle, true, true, true)
+            cc_utils.help_message(pid, "Vehicle repaired")
+        end
+    end
```
Comment (hexarobi) [inline lib/ChatCommands/vehicle/repair.lua:23]: anything changing on this file?

File: ChatCommander.lua (hunk 1 above)
Comment (hexarobi) [reply ChatCommander.lua:460]: I dont remember =) maybe its not needed

File: lib/ChatCommands/vehicle/repair.lua (hunk 2 above)
Comment (maximilianspitzer) [reply lib/ChatCommands/vehicle/repair.lua:23]: nope

Comment (maximilianspitzer) [reply lib/ChatCommands/vehicle/repair.lua:23]: > Sidenote: Im not sure why the !fix alias is still in this PR, even though there is no change to the file.

not sure why github thinks theres, changes, the files are 100% identical

**Pull request merged**